model/pod/db.go

```diff
@@ -65,3 +65,16 @@ func UpdateOne(query interface{}, update interface{}) error {
 		update,
 	)
 }
+
+// Find gets all Pods for the given query.
+func Find(query db.Q) ([]Pod, error) {
```

Comment:
Can you pass a `bson.M` instead of a `db.Q`? `bson.M` is more flexible when you want to combine queries together and it avoids having to call `db.Query` before every single call to `Find`. You can call `db.Query` within this function.